db/generate_data.js

```diff
@@ -10,7 +10,7 @@ let shoeBulkData = [];
 gender.forEach(gender => {
   material.forEach(material => {
     action.forEach(action => {
-      shoeBulkData.push({ name: `${gender} ${material} ${action}`, model: model });
```

Comment:
Why did you take out this model piece?

Reply:
I set the model column to be auto incrementing, so that when generating this data I don't have to manually define the model number

Reply:
If this were real life data the model numbers would definitely be defined here, but when creating dummy data like this I think that it is alreadt to auto generate it like this.\